packages/embark/src/lib/modules/deployment/index.js

```diff
@@ -107,7 +107,7 @@ class DeployManager {
               done(err);
             });
           }
-        ]);
+        ], (err) => err && self.logger.error(err));
```

Comment:
Actually, an even better idea would be to put `done` here and above, instead of using `done`, use `next` (which need to be added on line 64